cips/cip-shwap_protocol.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Shwap Protocol 
+description: Shwap - the new extensible sampling protocol 
```

Comment:
can the description explain what it is? Just one liner is fine. New extensible sampling protocol doesn't explain much. Something like "Shwap - a new efficient sampling protocol that combines shares and swaps" or something

Reply:
+

Reply:
This is resolved